adal/src/main/java/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -122,15 +121,15 @@
 @SuppressLint({
         "SetJavaScriptEnabled", "ClickableViewAccessibility"
 })
-public class AuthenticationActivity extends Activity {
+public class AuthenticationActivity extends DualScreenActivity {
 
     static final int BACK_PRESSED_CANCEL_DIALOG_STEPS = -2;
 
     private static final String TAG = "AuthenticationActivity";
     private boolean mRegisterReceiver = false;
     private WebView mWebView;
     private String mStartUrl;
-    private ProgressDialog mSpinner;
+    private ProgressBar mSpinner;
```

Comment:
Let's rename this property, since it's not a `Spinner` anymore

Reply:
It's still a spinner actually. Just not in a dialog box.

Reply:
@rpdome [Spinner](https://developer.android.com/guide/topics/ui/controls/spinner)'s a chooser box, not a progress indicator

Reply:
I guess the name was always wrong? 🤦‍♂

Reply:
ahhh.... what I meant is that it's a spinner as in... it's a spinning progress bar 🤣

Reply:
but yeah this has always been the name.